src/main/java/org/isf/visits/gui/InsertVisit.java

```diff
@@ -448,7 +448,7 @@ public void actionPerformed(ActionEvent e) {
 	private JPanel getPanelChoosePatient() {
 		JPanel choosePatientPanel = new JPanel();
 		choosePatientPanel.setLayout(new FlowLayout(FlowLayout.CENTER));
-		choosePatientPanel.setBorder(BorderFactory.createTitledBorder(MessageBundle.getMessage("angal.common.pleaseselectapatient"))); //$NON-NLS-1$
+		choosePatientPanel.setBorder(BorderFactory.createTitledBorder(MessageBundle.getMessage("angal.visit.pleaseselectapatient.border")));
```

Comment:
The "good" to have duplicated keys is to have all labels concerning the same package under the same root key and of course to provide specific translations for some of them (and also the original intention though); the "evil" si redundancy of texts and translations. 
Balancing between the two strategy is something we should document somewhere for devs, just a note!

Reply:
I am going to make another pass through the keys and handle `title` and `border` as mentioned in the other PR.

Reply:
ok